### ISSUE TITLE
Matrix server with Discord bridge bot

### ISSUE THREAD

**Issue by candroid-man** [streetwriters/notesnook#174]
**Request**

Create a [Matrix](https://matrix.org/) server, and link the Discord and Matrix server together using [matrix-appservice-discord](https://github.com/Half-Shot/matrix-appservice-discord) or [matrix-discord-bridge](https://github.com/git-bruh/matrix-discord-bridge). This would allow for people to get support and report bugs in a more private way, and the bots work very well. If you would like to test them out and see how they work, you can try it out in the [Techlore Discord server](https://discord.gg/V5XrKpZ8) or their [Matrix server](https://matrix.to/#/#techlore:techlore.net).

**Comment by trymeouteh** [comment]
Would like a matrix server

**Comment by vertigo220** [comment]
Agree, and GitHub and other contact links besides just Discord should be added to the "Other" section in settings. I almost just gave up and moved on since it looked like Discord was the only way to communicate, and I HATE Discord and am switching away from it, and an open-source app focused on privacy should focus on using other, better communication channels, like Matrix and XMPP. It's one thing to use Discord since a lot of people use it, but it shouldn't be the only, or arguably even the primary, one for an app like this. And I wouldn't want to pay for a pro account only to have to deal with email for support because I can't use Discord.
